### PR TITLE
Migrate aws sdk to v3 for serverless demo and react router to v6

### PR DESCRIPTION
**Pull request by xuesichao**
**Issue #:**

**Description of changes:**
* Migrate aws sdk from v2 to v3 for serverless handler
* Migrate react router from v5 to v6 to be compatible with react 18
* 
**Testing**

1. How did you test these changes?
    Deploy the serverless demo and tested e2e.
2. Can these changes be tested using one of the demo application? If yes, which demo application can be used to test it?
    Yes react meeting demo, deploy serverless demo and test e2e.

3. If applicable, have you run `npm run build` successfully locally to fix all warnings and errors?
    `npm run build` succeeded.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: apps/meeting/.eslintrc.json [hunk 1]
```diff
@@ -13,6 +12,9 @@
       "version": "detect"
     }
   },
+  "files": [
+    "src/*.(ts|tsx)"
```
Comment (xuesichao) [inline apps/meeting/.eslintrc.json:16]: Now our eslint now only targets the react app source code. It uses `@typescript-eslint` and was also targeting `.js` file mistakenly.

File: apps/meeting/serverless/template.yaml [hunk 2]
```diff
@@ -12,7 +12,7 @@ Conditions:
   ShouldUseEventBridge: !Equals [true, !Ref UseEventBridge]
 Globals:
   Function:
-    Runtime: nodejs14.x
+    Runtime: nodejs18.x
```
Comment (xuesichao) [inline apps/meeting/serverless/template.yaml:15]: Lambda is deprecating node14 now.

File: apps/meeting/serverless/template.yaml [hunk 3]
```diff
@@ -140,23 +139,6 @@ Resources:
           Properties:
             Path: /{proxy+}
             Method: ANY
-  ChimeSdkCreateMeetingLambda:
```
Comment (xuesichao) [inline apps/meeting/serverless/template.yaml:143]: `POST /createMeeting` is being used

**Review by michhyun1 — APPROVED** [review]

**Comment by michhyun1** [comment]
awesome job!

**Review by linsang21 — APPROVED** [review]

**Pull request merged**